workers/tasks/export/notes.js

```diff
@@ -0,0 +1,43 @@
+import { jsonToNode, getNotes } from 'components/Editor';
```

Comment:
This is the spiritual successor to the deleted `prosemirror.js`, but it only does notes-related things now.